src/compiler/builder.ts

```diff
@@ -1457,13 +1465,13 @@ function getBuildInfo(state: BuilderProgramStateWithDefinedProgram): BuildInfo {
         if (option) {
             Debug.assert(option.type !== "listOrElement");
             if (option.type === "list") {
-                const values = value as readonly string[];
+                const values = value as readonly unknown[];

```

Comment:
I actually think that this type assertion probably isn't correct, and we need to validate that `value` actually is an array.

Reply:
I couldn't get this to fail by writing a test.